s2spy/utils.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+import xarray as xr
+
+
+PandasData = (pd.Series, pd.DataFrame)
+XArrayData = (xr.DataArray, xr.Dataset)
+
+
+def check_input_data(data) -> None:
```

Comment:
If this is a generic `util` module, then perhaps this function name should be more specific, e.g. `check_timeseries` or so? Since there are many things you can check wrt input data. Or, if it's so specifically linked to `time.py`, perhaps it makes sense to create a s2s/time/utils.py specific utility folder (and move time.py and _base_calendar.py also to time/time.py).

Reply:
For now I have renamed the function. I'll add refactoring the module/folder structure as an item in an issue, as I do not want to mess with the current PR too much.